homeassistant/components/laundrify/strings.json

```diff
@@ -1,4 +1,19 @@
 {
+  "entity": {
+    "binary_sensor": {
+      "wash_cycle": {
+        "name": "Cycle"
+      }
+    },
```

Comment:
Unrelated change

Reply:
Moved to https://github.com/home-assistant/core/pull/121435 and undone in https://github.com/home-assistant/core/commit/6e0e80a4a44a3a07743e85438716c610ad64652e